addons/binding/org.openhab.binding.jeelink/README.md

```diff
@@ -0,0 +1,135 @@
+| Sensor Timeout    | Number       | The amount of time in seconds that should result in OFFLINE status when no readings have been received from the sensor
+| Update Interval   | Number       | The update interval in seconds how often state updates are propagated
+| Buffer Size       | Number       | The number of readings used for computing the rolling average
+| Lower Temperature Limit | Decimal       | The lowest allowed valid temperature. Lower temperature readings will be ignored
+| Upper Temperature Limit | Decimal       | The highest allowed valid temperature. Higher temperature readings will be ignored
+
+#### EC3000 power monitors
+
+| Parameter         | Item Type    | Description
+|-------------------|--------------|------------
+| Sensor ID         | Number       | The ID of the connected sensor
+| Sensor Timeout    | Number       | The amount of time in seconds that should result in OFFLINE status when no readings have been received from the sensor
+| Update Interval   | Number       | The update interval in seconds how often state updates are propagated
+| Buffer Size       | Number       | The number of readings used for computing the rolling average
+
+
+## Channels
+
+#### JeeLink USB RF receivers 
+
+Do not have any channels.
+
+#### LaCrosse temperature sensors
+
+| Channel Type ID         | Item Type    | Description
+|-------------------------|--------------|------------
+| sensorId                | -            | Only used for commanding
+| sensorType              | Number       | The sensor type
+| temperature             | Number       | Temperature reading
+| humidity                | Number       | Humidity reading 
+| batteryNew              | Contact      | Whether the battery is new (CLOSED) or not (OPEN)
+| batteryLow              | Contact      | Whether the battery is low (CLOSED) or not (OPEN)
+
+#### EC3000 power monitors
+
+| Channel Type ID         | Item Type    | Description
+|-------------------------|--------------|------------
+| sensorId                | -            | Only used for commanding
+| currentWatt             | Number       | Instantaneous power in Watt
+| maxWatt                 | Number       | Maximum load power in Watt
+| consumptionTotal        | Number       | Total energy  consumption 
+| applianceTime           | Number       | Total electrical appliance operating time in hours
+| sensorTime              | Number       | Total turn on time of power monitor in hours
+| resets                  | Number       | Number of resets
+
+## Commands
+
+#### JeeLink USB RF receivers
+
+Do not handle any commands.
+
+#### LaCrosse temperature sensors and EC3000 power monitors
+
+| Channel Type ID         | Command Type    | Command Syntax       | Description
+|-------------------------|-----------------|----------------------|------------
+| sensorId                | String          | SetSensorId _newID_  | Sets the sensor ID of the current sensor to the specified _newID_
+
+
```

Comment:
I have no idea how a things file looks like. All my things are created with the paper UI.